tool/README.md

```diff
@@ -0,0 +1,3 @@
+# excel-to-json-
+
+#website Link https://www.codesolution.co.in/
```

Comment:
Is this link still relevant for anything?

I do think this file could use a short description how to run the tool

Reply:
replaced it with a very short description how to run